includes/class-amp-story-post-type.php

```diff
@@ -93,20 +100,22 @@ class AMP_Story_Post_Type {
 	/**
 	 * Check if the required version of block capabilities available.
 	 *
-	 * Requires either Gutenberg 6.6+ or WordPress 5.3+ (which includes Gutenberg 6.6)
-	 *
-	 * @todo Eventually the Gutenberg requirement should be removed.
-	 *
 	 * @return bool Whether capabilities are available.
 	 */
 	public static function has_required_block_capabilities() {
-		return (
-			( defined( 'GUTENBERG_DEVELOPMENT_MODE' ) && GUTENBERG_DEVELOPMENT_MODE )
-			||
-			( defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, self::REQUIRED_GUTENBERG_VERSION, '>=' ) )
-			||
-			version_compare( get_bloginfo( 'version' ), '5.3-RC2', '>=' )
-		);
+		if ( defined( 'GUTENBERG_DEVELOPMENT_MODE' ) ) {
+			return (bool) GUTENBERG_DEVELOPMENT_MODE;
```

Comment:
Wouldn't that indicate that the editor works with current Gutenberg master?

I think if `GUTENBERG_DEVELOPMENT_MODE` is defined we should return `false` as it's most likely the current master version.

Reply:
Yes good call.

Would it be better to get the version via `get_plugin_data()`? The likelihood of someone having an outdated Gutenberg with `GUTENBERG_DEVELOPMENT_MODE` set would be low I assume, but I'm not entirely on board with returning `false` if that constant is set.

Reply:
Let's go ahead and return `false` if `GUTENBERG_DEVELOPMENT_MODE` is on. The reality is that this is not a widely-used feature yet and we'll be removing it in the next version of the AMP plugin. The liklihood that someone is using an old version of Gutenberg _with_  `GUTENBERG_DEVELOPMENT_MODE` on _and_ having Stories active seems very rare. If `GUTENBERG_DEVELOPMENT_MODE` is `true`, then a site just has to deactivate Gutenberg to access Stories.

Reply:
Done in https://github.com/ampproject/amp-wp/pull/4217/commits/92e7c3481444a40801b439e94c874886f145afa1.